### ISSUE TITLE
Application is not working with standalone backend

### ISSUE THREAD

**Issue by 12masta** [gothinkster/react-redux-realworld-example-app#141]
I am facing issue when trying to run app with local backend. If someone can help I will be very grateful. Steps to reproduce:

**Preconditions**:

- local running build of https://github.com/gothinkster/aspnetcore-realworld-example-app in my case server is running locally on: http://localhost:5000/api

**Steps to reproduce:**

1. Run backend
2. Edit src/agent.js and change API_ROOT to the local server's URL, in my case: http://localhost:5000/api
3. npm install and npm start in project root directory

**Actual results:**
* After successful npm start browser is bringing up with following error: Unhandled Rejection (TypeError): Cannot read property '0' of null

**Expected results:**
* Successful build and properly running web application

**Comment by 12masta** [comment]
Tldr:
replace api url in agent.js with **http://localhost:5000** instead of http://localhost:5000/api

I think there is a bug in repo description, you can find following line:

> If you want to change the API URL to a local server, simply edit src/agent.js and change API_ROOT to the local server's URL (i.e. http://localhost:3000/api)

But in this case web app trying to reach for tag endpoint with following url: http://localhost:3000/api/tags 

Which is not valid, because tags endpoint is exposed under this url: 
http://localhost:3000/tags
